### ISSUE TITLE
0.3.0 failing when sending lcov data 

### ISSUE THREAD

**Issue by laynegt** [codeclimate/javascript-test-reporter#33]
Get this in 0.3.0 when sending lcov data.  Works in 0.2.1.

```
/home/rof/.nvm/versions/node/v0.12.0/lib/node_modules/codeclimate-test-reporter/formatter.js:64
data.forEach(function(elem, index) {
^
TypeError: Cannot read property 'forEach' of null
at Formatter.sourceFiles (/home/rof/.nvm/versions/node/v0.12.0/lib/node_modules/codeclimate-test-reporter/formatter.js:64:7)
at /home/rof/.nvm/versions/node/v0.12.0/lib/node_modules/codeclimate-test-reporter/formatter.js:33:26
at Formatter.parse (/home/rof/.nvm/versions/node/v0.12.0/lib/node_modules/codeclimate-test-reporter/formatter.js:24:9)
at Formatter.format (/home/rof/.nvm/versions/node/v0.12.0/lib/node_modules/codeclimate-test-reporter/formatter.js:31:8)
at ReadStream.<anonymous> (/home/rof/.nvm/versions/node/v0.12.0/lib/node_modules/codeclimate-test-reporter/bin/codeclimate.js:25:13)
at ReadStream.emit (events.js:129:20)
at _stream_readable.js:908:16
at process._tickCallback (node.js:355:11)
```

Don't think it's related to https://github.com/codeclimate/javascript-test-reporter/issues/16, my lcov file is populated.

**Comment by oniofchaos** [comment]
I'm having the same issue. Downgraded to 0.2.1 and it works again.

**Comment by devinus** [comment]
Same issue.

**Comment by GordonDiggs** [comment]
@devinus @oniofchaos @laynegt Sorry about that! #34 should fix it. I just released version 0.3.1 to npm
